GVFS/GVFS.Common/NuGetUpgrader/InstallManifestPlatform.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GVFS.Common.NuGetUpgrader
+{
+    public class InstallManifestPlatform

```

Comment:
I would remove this class because it is not giving any value right now.  If in the future we need something that does more we can add it back in.